inst/rmarkdown/templates/gigascience_article/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,548 @@
+\caption{Table generated using rmarkdown}
+\label{tab:markdown}
+```{r setup, echo=FALSE, warning=FALSE}
+library(knitr)
+knitr::opts_chunk$set(echo = TRUE)
+
+kable(
+  t(
+    data.frame(
+      header=c("Right","Left","Center")
+    , value=c(1,2,3) 
+    )
+  )
+, align = c("r","l","c")
+, format = "latex")
+```
+\end{table}
+
+### Laying out tables in \LaTeX{}
+
+Table layout in \LaTeX{} is much less intuitive than using R, but this might be your only choice unless you define a section that has only one column per page.
+
+
+
+## Pre-existing figure
+
+It is inadvisable to generate images from R code chunks, but it's fine to include pre-existing images.
+
+```{r warning=FALSE, dev='pdf', fig.show='hide', echo=FALSE, message=FALSE}
+# That being said, a file is being generated here solely for demonstration purposes.
+invisible({
+  pdf(file=file.path(getwd(),'pressure.pdf'))
+  plot(pressure)
+  dev.off()
+})
+```
+
+### Including a figure using \LaTeX{}
+
+The easier way to include a figure is with \LaTeX{}.
+Figure \ref{fig:pressure} is included this way, and the figure number is a clickable reference.
+```
+% "bt!" means favor putting the table at the 
+%       top or bottom of the column, if possible
+% "b!"  means favor putting the table at the 
+%       bottom of the column, if possible
+% "t!"  means favor putting the table at the 
+%       top of the column, if possible
+\begin{figure}[bt!]
+\centering
+\includegraphics[width=\linewidth]{pressure.pdf}
+\caption{Pressure figure from \LaTeX{}}
+\label{fig:pressure}
+\end{figure}
+```
+\begin{figure}[bt!]
+\centering
+\includegraphics[width=\linewidth]{pressure.pdf}
+\caption{Pressure figure from \LaTeX{}}
+\label{fig:pressure}
+\end{figure}
+
+### Including a figure using markdown
+
+The "Pressure from markdown"" figure is scaled 48% because there are two columns, and failing to scale the image results in an image that is two columns wide; when scaled to 50%, it does not quite fit, so the scaling value needed to be determined emprically.  Furthermore, no clickable link to it is available becasue the generated TeX code has no label attribute.  In fact, to get the figure number, it would probably be necessary to resort to using a \LaTeX{} expression anyway....
+```
+![Pressure figure from markdown](pressure.pdf){width=48%}
+```
+![Pressure figure from markdown](pressure.pdf){width=48%}
+
+Overall, it's easier to use \LaTeX{} code than force rmarkdown to do the task.
+
+```{r refs_section_label, echo=FALSE}
+  # \bibliography generates the References section label and makes the bibliography entries available through natbib to citep and citet
+  # However, it does not do so when running knit on .Rmd from within RStudio.
+  #   The workaround is to open the resulting .tex file and click 'Compile PDF' a few times until there is no error.
+```
+
+## Citations in rmarkdown (not recommended)
+
+Using rmarkdown for citations is not recommended: 
+  - TeX citations give you more flexibility
```

Comment:
For example? What kind of flexibility does an average author need?

Reply:
Bibliography-generated automated in TeX so that edits made in proof (on Overleaf) will not adversely impact the bibliography.